src/org/jitsi/impl/neomedia/RTPConnectorOutputStream.java

```diff
@@ -768,11 +768,19 @@ public long getOutputBitrate(long now)
          */
         final Thread sendThread;
 
+        QueueStatistics queueStats = null;
```

Comment:
missing JavaDocs ? also why does it have "package private" access level ? Just asking...